spec/factories/developer_languages.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :developer_language do
+    developer_id 1
```

Comment:
You should define relationship, like `developer`, not a number.
http://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md

You shouldn't define a number in an intermediate table directly.
Because it is the number of an actual table (the developers table) in this case.
But you can't tell it's actually `1`.
Hope the explanation makes sense. (I'm not sure if I can explain it in English appropriately)